lib/adal.js

```diff
@@ -22,29 +22,23 @@ var AuthenticationContext = (function () {
     'use strict';
 
     /**
-     * Config information
-     * @public
-     * @class Config
-     * @property {tenant}          Your target tenant
-     * @property {clientId}        Identifier assigned to your app by Azure Active Directory
-     * @property {redirectUri}     Endpoint at which you expect to receive tokens
-     * @property {instance}        Azure Active Directory Instance(default:https://login.microsoftonline.com/)
-     * @property {endpoints}       Collection of {Endpoint-ResourceId} used for autmatically attaching tokens in webApi calls
-     */
-
-    /**
-     * User information from idtoken.
-     *  @class User
-     *  @property {string} userName - username assigned from upn or email.
-     *  @property {object} profile - properties parsed from idtoken.
+     * Configuration options for Authentication Context.
+     * @class config
```

Comment:
We are missing some public properties on this class: callback, displayCall, anonymousEndpoints, expireOffsetSeconds, logOutUri, postLogoutRedirectUri, cacheLocation, correlationId. We can skip `isAngular` as we don't want users to use it for plain JS.